### PR TITLE
add verify job step for image locations

### PR DESCRIPTION
**Pull request by dhellmann**
Related to [USHIFT-499](https://issues.redhat.com//browse/USHIFT-499)

### CODE REVIEW

File: scripts/verify_images.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+function debug() {
+    if ! $verbose; then
+        return
+    fi
+    echo "$*"
+}
+
+approved=true
+./pkg/release/get.sh images all | while read source_file image; do
```
Comment (ggiguash) [inline scripts/verify_images.sh:19]: Can we change this from relative to $ROOTDIR?

ROOTDIR=$(git rev-parse --show-toplevel)

File: scripts/verify_images.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+verbose=false
+if [ "$1" = "-v" ]; then
+    verbose=true
+fi
+
+function debug() {
+    if ! $verbose; then
+        return
+    fi
+    echo "$*"
```
Comment (ggiguash) [inline scripts/verify_images.sh:15]:
```suggestion
    $verbose && echo "$*"
```

**Review by ggiguash — CHANGES_REQUESTED** [review]

File: scripts/verify_images.sh (hunk 1 above)
Comment (dhellmann) [reply scripts/verify_images.sh:16]: Done

**Comment by dhellmann** [comment]
@ggiguash I think we can go ahead with this as long as the verify job itself passes, since there shouldn't be any other impact.

**Comment by pmtk** [comment]
/test e2e-openshift-conformance-sig-storage
looks like gcp connection flakiness

**Comment by pmtk** [comment]
> @dhellmann: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> ci/prow/periodic-ocp-4.13-images 	[39e80fa](https://github.com/openshift/microshift/commit/39e80faffd099c9071a7c070b53326dac9951776) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_microshift/1043/pull-ci-openshift-microshift-main-periodic-ocp-4.13-images/1585221942107967488) 	true 	`/test periodic-ocp-4.13-images`
> 
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openshift&repo=microshift&pr=1043). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:dhellmann).

I don't see it in GH status, this can be ignored (esp it's for commit pre rebase)

**Comment by dhellmann** [comment]
> > @dhellmann: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> > Test name 	Commit 	Details 	Required 	Rerun command
> > ci/prow/periodic-ocp-4.13-images 	[39e80fa](https://github.com/openshift/microshift/commit/39e80faffd099c9071a7c070b53326dac9951776) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_microshift/1043/pull-ci-openshift-microshift-main-periodic-ocp-4.13-images/1585221942107967488) 	true 	`/test periodic-ocp-4.13-images`
> > [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openshift&repo=microshift&pr=1043). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:dhellmann).
> 
> I don't see it in GH status, this can be ignored (esp it's for commit pre rebase)

@pmtk I think that was the job added mistakenly and then removed, right? Either way, as you say we should be able to proceed with this.

**Pull request merged**